src/sentry/templates/sentry/plugins/bases/issue/create_issue.html

```diff
@@ -12,6 +12,8 @@
 
 {% block scripts %}
   {{ block.super }}
+  {% asset_url 'sentry' 'vendor/jquery.2.2.4.min.js' as jquery_url %}
```

Comment:
We're really close to being able to totally remove this from the SPA app I think